apps/easypid/src/use-cases/ReceivePidUseCaseFlow.ts

```diff
@@ -1,13 +1,13 @@
 import { AusweisAuthFlow, type AusweisAuthFlowOptions, sendCommand } from '@animo-id/expo-ausweis-sdk'
 import type { MdocRecord } from '@credo-ts/core'
 import type { AppAgent } from '@easypid/agent'
-import {
-  type OpenId4VciRequestTokenResponse,
-  type OpenId4VciResolvedCredentialOffer,
-  type OpenId4VciResolvedOauth2RedirectAuthorizationRequest,
-  type SdJwtVcRecord,
-  acquireAuthorizationCodeAccessToken,
+import type {
+  OpenId4VciRequestTokenResponse,
+  OpenId4VciResolvedCredentialOffer,
+  OpenId4VciResolvedOauth2RedirectAuthorizationRequest,
+  SdJwtVcRecord,
 } from '@package/agent'
+import { acquireAuthorizationCodeAccessToken } from '@package/agent/src/invitation/handler'
```

Comment:
Yes because it's local build with expo. So the packages boundaries don't really matter here.

It's a but ugly and will refactor after the deadline (had to prevent require cyccle)